markdown/overview.md

```diff
@@ -4,12 +4,19 @@ XStream has four fundamental types: Stream, Listener, Producer, and MemoryStream
 
 ## Stream
 
-A Stream is an **event emitter** with multiple Listeners. When an event happens on the
-Stream, it is broadcast to all its Listeners at the same time.
+A Stream is an *event emitter*. It can have any number of *listeners*. When an event occurs on the
+Stream, it is broadcast to all its *listeners*, synchronously and sequentially.
```

Comment:
Is it true that it is synchronously and sequentially? Perhaps the term sequentially is redundant?